client/components/Routes.js

```diff
@@ -0,0 +1,46 @@
+import React from "react"
+import { HashRouter as Router, Route , Link, Switch} from 'react-router-dom'
+import Home from './Home'
+import SingleUser from './SingleUser'
+import Login from './FormikLogin'
+
+export default class Routes extends React.Component {
+	constructor() {
+		super()
+		this.state = {
+			userId: 0
+		}
+	}
+
+	componentDidMount() {
+		this.setState({userId: 1})
+	}
+
+	render(){
+		return (
+			<Router>
+				<div>
+					<nav>
+						<Link className="navbar" to = "/">Home</Link>
+						<Link className="navbar" to = {`/users/${this.state.userId}`}>View Profile</Link>
```

Comment:
instead of mixing up all the authorized routes with the unauthorized routes, you can use a Switch (which kinda works just like a switch statement) 

example: 
```suggestion
import { Switch } from 'react-router-dom';

<Switch>
        {/* Routes placed here are available to all visitors */}
        {isLoggedIn && (
          <Switch>
            {/* Routes placed here are only available after logging in */}
          </Switch>
        )}
        {/* Displays our Login component as a fallback */}
</Switch>
```